### DISCUSSION TITLE
blog/compare-two-task-queues

### DISCUSSION THREAD

**Discussion by giscus** [yunjeoming/yuns-blog#60] [Comments]
# blog/compare-two-task-queues

두 태스크 큐의 차이점은? 🧐

https://yunjeoming.dev/blog/compare-two-task-queues

**Comment by NukeKang** [comment]
태스크 큐 vs 마이크로 태스크 큐
둘 다 큐(Queue) 자료 구조를 사용하지만..
이라고 하셨는데 태스크큐는 큐가 아니라 set 형태를 띠고 있습니다.
